### PR TITLE
EAOC-12300: Dry run & inspect replay session(Hugo)

### PR DESCRIPTION
**Pull request by HugoTealeaf**
+ Updates to latest library versions

### CODE REVIEW

File: Sunflower/build.gradle [hunk 1]
```diff
@@ -18,38 +18,38 @@ buildscript {
     // Define versions in a single place
     ext {
         // Sdk and tools
-        compileSdkVersion = 31
+        compileSdkVersion = 33
```
Comment (KJSweeneyMetova) [inline Sunflower/build.gradle:21]: the sdks compile against 32 - do we need to have the sample app at the same version?

**Review by KJSweeneyMetova — APPROVED** [review]
minor comment

File: Sunflower/build.gradle (hunk 1 above)
Comment (HugoTealeaf) [reply Sunflower/build.gradle:21]: I originally updated to 32, but then one of the updated Android libraries required compiling with 33.

**Review by cyangibm — APPROVED** [review]
Looks good, think it's ok for this app to target 33.  We'll upgrade to 33 for all apps next release.

**Pull request merged**